assets/javascripts/discourse/initializers/discourse-calendar.js

```diff
@@ -30,6 +33,8 @@ function initializeDiscourseCalendar(api) {
     return;
   }
 
+  let enableTimezoneOffset = siteSettings.default_timezone_offset_user_option;
```

Comment:
Do we have too many site settings related to this feature now? Maybe worth a followup PR.

Reply:
I wasn't thinking of adding this one, but since it takes a while to rerender the events when clicking the button, it makes it more reliable for testing. Maybe aother way could be to await for the the button state or something.
Yes, I'll probably have a followup PR revising all of this